src/constants.ts

```diff
@@ -38,6 +38,7 @@ export const Urls = {
     INDIVIDUAL_PSC_LIST: "/individual/psc-list",
     NEW_SUBMISSION: "/new-submission",
     PERSONAL_CODE: `${urlWithTransactionIdAndSubmissionId}/individual/personal-code`,
+    NAME_MISMATCH: `${urlWithTransactionIdAndSubmissionId}/individual/name-mismatch`,
```

Comment:
I wonder if we should be following the prototype here and use the path `/individual/psc-why-this-name` ?